src/endpoints/Coverage/Client.ts

```diff
@@ -100,7 +100,7 @@ export function createClient(api: DeferredJobsApiClient) {
         return api.delete(`${routing.coverageUrl}/${id}`);
     }
 
-    async function bulkDelete(
```

Comment:
Problem 4: `async` keywords was converting the returned `ProgressPromise` object to a regular `Promise`